src/App.css

```diff
@@ -32,28 +46,229 @@ body {
 }
 
 .book-wrapper {
+  display: flex;
   border: solid 1px #e8e8e8;
   background-color: #fff;
   width: 75rem;
   min-height: 10.625rem;
+  align-items: center;
+}
+
+.p-container {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.p-completed {
+  display: flex;
   flex-direction: column;
-  justify-content: center;
-  align-items: flex-start;
+  align-items: center;
+}
+
+.percent-no {
+  width: 4.25rem;
+  height: 2.438rem;
+  margin: 0.063rem 5.75rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.completed {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.progress {
+  width: 4.25rem;
+  height: 4.25rem;
+}
+
+.Current-Chapter {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.Current-Lesson {
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.update-progress {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  border: none;
+  background-color: #0290ff;
+  color: white;
+}
+
+.progress-wrapper {
+  display: flex;
+}
+
+.chapter-wrapper {
+  border-left: 1px solid #e8e8e8;
+}
+
+.form-header {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+  text-transform: uppercase;
+}
+
+.book-title-input {
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  padding: 0.8rem;
+  width: 25rem;
+  color: #989898;
+  margin-right: 2rem;
+  font-size: 1rem;
+}
+
+.form-wrapper {
+  margin-left: 3rem;
+}
+
+.add-btn {
+  width: 11.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+  color: #fff;
+}
+
+.book-title-input:focus {
+  outline: none;
 }
 
 .booklist {
   flex-direction: column;
   gap: 0.938rem;
 }
 
-.remove-button {
-  width: 4.375rem;
-  padding: 5px;
+.category {
+  width: 3.063rem;
+  height: 1.125rem;
```

Comment:
- [ ] `optional`
- [ ] please make the width which are `marked` in the screenshot below. that they can all equal sides as the original design.

![image](https://user-images.githubusercontent.com/66230306/201338540-60f5bd25-dafc-455a-b304-1334f866d6c2.png)